sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/BridgeInternal.java

```diff
@@ -52,6 +53,10 @@ public static Document documentFromObject(Object document, ObjectMapper mapper)
         return Document.FromObject(document, mapper);
     }
 
+    public static void monitorTelemetry(MeterRegistry registry) {
```

Comment:
⚠ This is public API change exposing an unapproved public dependency, but we've documented that this public API shouldn't be depended on, so I'm not seeking approval at this time. Approved.